api/get_commitment_test.go

```diff
@@ -105,6 +110,47 @@ func (s *GetCommitmentTestSuite) SetupTest() {
 		},
 	}
 
+	s.transfer = &models.Transfer{
+		TransactionBase: models.TransactionBase{
+			Hash:         utils.RandomHash(),
+			TxType:       txtype.Transfer,
+			FromStateID:  1,
+			Amount:       models.MakeUint256(50),
+			Fee:          models.MakeUint256(10),
+			Nonce:        models.MakeUint256(0),
+			Signature:    models.MakeRandomSignature(),
+			CommitmentID: &s.txCommitment.ID,
+		},
+		ToStateID: 2,
+	}
+
+	s.create2transfer = &models.Create2Transfer{
+		TransactionBase: models.TransactionBase{
+			Hash:         utils.RandomHash(),
+			TxType:       txtype.Create2Transfer,
+			FromStateID:  1,
+			Amount:       models.MakeUint256(50),
+			Fee:          models.MakeUint256(10),
+			Nonce:        models.MakeUint256(0),
+			CommitmentID: &s.txCommitment.ID,
+		},
+		ToStateID:   ref.Uint32(2),
+		ToPublicKey: models.PublicKey{2, 3, 4},
+	}
+
+	s.massMigration = &models.MassMigration{
+		TransactionBase: models.TransactionBase{
+			Hash:         utils.RandomHash(),
+			TxType:       txtype.MassMigration,
+			FromStateID:  1,
+			Amount:       models.MakeUint256(50),
+			Fee:          models.MakeUint256(10),
+			Nonce:        models.MakeUint256(0),
+			CommitmentID: &s.mmCommitment.ID,
+		},
+		SpokeID: 2,
+	}
+
```

Comment:
Maybe extract and call `setTxs()` method instead and/or use `testutils` helper functions.

Reply:
Done. I would leave further refactoring of similar problem with commitments in this `SetupTest` method for `PRO-194`.